wadas/ui/configure_db_dialog.py

```diff
@@ -81,6 +84,7 @@ def init_dialog(self):
     def on_radioButton_checked(self):
         """Method to update dialog fields depending on DB selection"""
 
+        self.ui.lineEdit_db_port.setEnabled(not self.ui.radioButton_SQLite.isChecked())
         self.ui.lineEdit_db_username.setEnabled(not self.ui.radioButton_SQLite.isChecked())
         self.ui.lineEdit_db_password.setEnabled(not self.ui.radioButton_SQLite.isChecked())
         self.ui.lineEdit_db_name.setEnabled(not self.ui.radioButton_SQLite.isChecked())
```

Comment:
Maybe you can  cache self.ui.radioButton_SQLite.isChecked() instead of always querying Qt for its status.
Also on other code with similar pattern.